eta-meeting-organizer-frontend/tslint.json

```diff
@@ -59,6 +59,7 @@
     // https://palantir.github.io/tslint/rules/no-parameter-properties/
```

Comment:
Ez mit takar?

Reply:
Kószó Tomi adott engedélyt, hogy rakjuk be az alábbi sort a Linter szabályok közé:
"object-literal-sort-keys": false Enélkül az Object kulcsoknak is abc sorrendben kellett lennie, és ez problémát okozott a másik csapatnál amikor az end-time a start-time elé került. Itt igazából csak abc sorrend szerint beraktam az alábbi sort.